framework/source/class/qx/test/Interface.js

```diff
@@ -365,7 +363,7 @@ qx.Class.define("qx.test.Interface",
             members :
             {
               getValue : function() {},
-              setValue : function(value) {}
```

Comment:
No ESLint complained (i used ESLint to fix those files and run the generator after).